content/try-opensds/installation-using-ansible.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Installation Using ansible
+description: "This is a  guide for users and new contributors to install Local Cluster using ansible."
+weight: 30
+---
+This document describes how to install SODA projects local cluster, including Hotpot (API, Controller, Dock), Gelato, Telemetry, Orchestration, and Dashboard components.
+
+`Hereafter, Hotpot refers to API, Controller and Dock projects`
+
+### Pre-config (Ubuntu 16.04)
+All the installation work is tested on `Ubuntu 16.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+
+<br /> Install following packages:
+
+```bash
+apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk
+```
+<br />Install docker:
+
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+```
+```bash
+dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+```
+<br /> Install docker-compose:
+
+```bash
+curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+```
+<br /> Install golang
+
+```bash
+wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
+tar -C /usr/local -xzf go1.12.1.linux-amd64.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile
+echo 'export GOPATH=$HOME/gopath' >> /etc/profile
+source /etc/profile
+```
+`Check if golang exists`
+```bash
+go version
+```
+<br />It should be go1.12.1 linux/amd64
+
+### Download SODA installer code
+```bash
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the latest stable release. Current stable release: stable/elba. If you want to get the master branch of all components, you can skip this step. (<span style="color:red"> Attn:</span> Master may not be stable or tested fully)
+git checkout v0.12.0
+```
+
+<br /> Install ansible tool
+<br />
+
+This step is needed to upgrade ansible to version 2.4.2 which is required for the "include_tasks" ansible command.
+<br />
+
+```bash
+chmod +x ./install_ansible.sh && ./install_ansible.sh
+ansible --version # Ansible version 2.4.x is required.
+```
+<br />Configure soda hotpot install variables:
+
+Firstly you need to modify host_ip in `group_vars/common.yml`, and you can specify which project (hotpot or gelato) to be deployed:
```

Comment:
Sentence structure.
**Firstly you need to modify host_ip...**  -> **Modify host_ip and change it to the actual machine IP of the host. You can specify which projects to deploy; only hotpot, only gelato or all**
The project names are going to change. Please change later when the names are finalised. For now we use hotpot and gelato.